learning-objectives/intl/es.yml

```
@@ -961,13 +961,13 @@ python/flask/request/args:
   title: Argumentos
   links:
     - title: Request - Flask Docs (en inglés)
-      url: https://flask.palletsprojects.com/en/2.0.x/quickstart/#accessing-request-data
+      url: https://flask.palletsprojects.com/en/3.0.x/quickstart/#accessing-request-data
 
 python/flask/request/headers:
   title: Cabeceras
```

```suggestion
  title: Headers
```
Probably is spanglish but I've never heard developers say: Cabeceras.

---

Ah ok, es que robe de https://github.com/Laboratoria/curriculum/blob/main/learning-objectives/intl/es.yml#L463
voy a cambiar